crates/host/src/wasmbus/mod.rs

```diff
@@ -794,13 +807,12 @@ impl Lattice {
         Ok(entry.insert(actor))
     }
 
-    #[instrument(skip(host_id))]
+    #[instrument(skip(self))]
     async fn stop_actor<'a>(
         &self,
         entry: hash_map::OccupiedEntry<'a, String, Arc<Actor>>,
-        host_id: impl AsRef<str>,
+        host_id: &str,
```

Comment:
why?

Reply:
As far as I can tell looking at the code, everything calling this private function has the ability to just use a normal `str` (or deref-ed `String`). Plus it has the added advantage of simplifying the logging